oak_runtime/src/node/mod.rs

```diff
@@ -32,10 +32,13 @@ mod wasm;
 
 /// Trait encapsulating execution of a Node or pseudo-Node.
 pub trait Node: Send {
-    /// Return a name for this type of Node.
+    /// Returns a name for this type of Node.
     fn node_type(&self) -> &'static str;
 
-    /// Execute the Node, using the provided `Runtime` reference and initial handle.  The method
+    /// Returns a value indicating whether the Node can communicate externally.
```

Comment:
Could you add more details to the comment? It seems an important parameter to choose, and we should make sure we don't get it wrong. In fact I would try to make it sound scarier, e.g. unsafe or uncontrolled or privileged or something like that. WDYT?

BTW personally I think it may read better as an enum with meaningfully named variants, e.g. sandboxed / privileged or internal / external or something else. Up to you (I expect we will come back to this later on once we have all thought about it anyways).

Reply:
Added more details and changed to an enum.